setup.py

```diff
@@ -28,7 +28,7 @@
     version="0.4.0",
     url="https://github.com/mirumee/ariadne",
     packages=["ariadne"],
-    package_data={"ariadne": ["py.typed"]},
+    package_data={"ariadne": ["py.typed", "contrib/**/templates/**/*.html"]},
```

Comment:
This pattern feels oddly specific - yet not at the same time. Could we just match all `.html` files within `ariadne`? That will include all potential html templates that may be added, and save us silly mistakes when we move things around and forget to update those paths.